.travis.yml

```diff
@@ -1,10 +1,20 @@
-sudo: false
+sudo: true
 
 language: java
 jdk: oraclejdk8
 
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - wget
+      - pkg-config
+
 before_install:
   - unset _JAVA_OPTIONS
+  - wget https://github.com/bazelbuild/bazel/releases/download/0.16.0/bazel_0.16.0-linux-x86_64.deb
+  - sudo dpkg -i bazel_0.16.0-linux-x86_64.deb
```

Comment:
Move the download and install command into a dedicated script, like `junit5-jupiter-starter-bazel/install-bazel.sh`, and call it only in the **"Jupiter Starter Sample: Bazel"** project. Bazel is only needed here.

`script: cd $TRAVIS_BUILD_DIR/junit5-jupiter-starter-bazel && ./install-bazel.sh && bazel test //...`

Reply:
I can't speak for the JUnit team, but that sounds rather reasonable to me @hcoona. :)

Reply:
Sure, I think it reasonable. But it require sudo in my script to install bazel, which would break the original travis ci container based infrastructure. Instead, put it to `apt` section would benifit from it, which, in my mind, would not take time for the installation in the future.